test/e2e/testdata/acme/clusterissuer.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt-prod
+spec:
+  acme:
+    email: ytripath@redhat.com
```

Comment:
Could we somehow use either a fake email or an alias to our team? @s-urbaniak WDYT?

Reply:
Should I rewrite this commit without my mail address, until then?

Reply:
Yeah, I think that would be better. But let's wait for @s-urbaniak 's opinion here.

Reply:
for tests, fake email sgtm

Reply:
Email has to be a valid one otherwise letsencrypt ACME server does not issue a certificate, we can use an alias, the only mails we will recieve will  be when the certificate is about to expire